Makefile

```diff
@@ -275,7 +275,7 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERS
 	| $(BUILD_DIRECTORY)
 	./scripts/build/concatenate-javascript.sh -e lib/cli/etcher.js -b $< -o $@ -m
 
-$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH): \
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH): \
```

Comment:
I've no opinion on which is "correct", but https://github.com/electron/electron/releases uses `name-version-platform-arch` and https://etcher.io uses `name-version-platform-arch` and https://etcher.io/cli/ uses `name-version-platform-arch`, so it probably makes sense to stick to that convention here in the `Makefile`? ;)